src/store.js

```diff
@@ -271,7 +278,9 @@ const store = new Vuex.Store({
   },
   actions: {
     /* Called when app first loaded. Reads config and sets state */
-    [INITIALIZE_CONFIG]({ commit }) {
+    async [INITIALIZE_CONFIG]({ commit }) {
+      // Get the config file from the server and store it for use by the accumulator
+      commit(SET_REMOTE_CONFIG, (await axios.get(serviceEndpoints.getConf)).data);
```

Comment:
I _think_ it should be possible to do this client-side, instead of having a server-side endpoint.
Since config is in the public directory, something like: `axios.get('/conf.yml').data`

That would mean the app would still work fine when run as a static web app. Right now, the config isn't available for anyone serving up `dist/` with their own server, or using a platform like Netlify  (demo: https://deploy-preview-528--dashy-dev.netlify.app/)